apps/condo/domains/ticket/tasks/reopenDeferredTickets.js

```diff
@@ -38,13 +41,38 @@ const reopenDeferredTickets = async () => {
         // TODO(DOMA-4155): Update ticket can break on getOrCreateContactByClientData
         for (const ticket of ticketsToChange) {
             try {
-                await Ticket.update(adminContext, ticket.id, {
+                const updatedData = {
                     dv: 1,
                     sender: { fingerprint: 'auto-reopen', dv: 1 },
-                    executor: { disconnectAll: true },
-                    assignee: { disconnectAll: true },
                     status: { connect: { id: STATUS_IDS.OPEN } },
-                })
+                    assignee: { disconnectAll: true },
+                    executor: { disconnectAll: true },
+                }
+                const assigneeId = get(ticket, ['assignee', 'id'])
+                const executorId = get(ticket, ['executor', 'id'])
+                const organizationId = get(ticket, ['organization', 'id'])
+
+                const employeeIds = []
+                assigneeId ? employeeIds.push(assigneeId) : delete updatedData.assignee
```

Comment:
please don't user ternary like that, it's anti-pattern

Reply:
refactored